ideascube/serveradmin/management/commands/catalog.py

```diff
@@ -0,0 +1,148 @@
+import argparse
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from ideascube.serveradmin.catalog import NoSuchPackage, Catalog
+
+
+class Command(BaseCommand):
+    help = 'Manage apps and content'
+
+    def add_arguments(self, parser):
```

Comment:
Not sure if it's easy to do, but could be nice to display a nice full help message instead of an error when calling "manage.py catalog" without any command. Maybe adding an "help" command and setting it as default?